src/components/Main/Mitigation/MitigationTable.tsx

```diff
@@ -117,44 +112,45 @@ function MitigationIntervalComponent({
   })
 
   return (
-    <FormGroup>
-      <div
-        className={`mitigation-interval ${
-          // eslint-disable-next-line unicorn/no-nested-ternary
-          width && width <= 325 ? 'very-narrow' : width && width <= 580 ? 'narrow' : 'wide'
-        }`}
-      >
-        <div className="inputs">
+    <FormGroup className="w-100 p-2">
+      <Row noGutters style={{ background: '#ddd' }}>
+        <Col xs="12" sm="12" md="12" lg="12" xl="12">
+          <InputGroup className="align-items-center p-1 pr-1">
+            <MitigationDatePicker
+              identifier={`containment.mitigationIntervals[${index}].timeRange`}
+              value={interval.timeRange}
+              allowPast
+            />
+            <Button type="button" onClick={() => arrayHelpers.remove(index)} className="ml-auto">
+              <FaTrash />
+            </Button>
+          </InputGroup>
+        </Col>
+        <Col xs="12" sm="12" md="12" lg="12" xl="12" className="align-self-center pl-1 pr-1">
           <FastField
             className={`name form-control ${nameError ? 'border-danger' : ''}`}
             id={`containment.mitigationIntervals[${index}].name`}
             name={`containment.mitigationIntervals[${index}].name`}
             type="text"
           />
-          <div className="item-date-range-value-group">
-            <MitigationDatePicker
-              identifier={`containment.mitigationIntervals[${index}].timeRange`}
-              value={interval.timeRange}
-              allowPast
-            />
-            <FastField
-              className={`form-control item-value ${valueError ? 'border-danger' : ''}`}
-              id={`containment.mitigationIntervals[${index}].mitigationValue`}
-              name={`containment.mitigationIntervals[${index}].mitigationValue`}
-              type="number"
-            />
-          </div>
-        </div>
-        <div className="item-controls">
-          <Button type="button" onClick={() => arrayHelpers.remove(index)}>
-            <FaTrash />
-          </Button>
-        </div>
-      </div>
-      <div className="w-100">
-        {nameError && <p className="my-0 text-right text-danger">{`${t('Intervention name')}: ${nameError}`}</p>}
-        {valueError && <p className="my-0 text-right text-danger">{`${t('Mitigation strength')}: ${valueError}`}</p>}
-      </div>
+        </Col>
+        <Col xs="12" sm="12" md="12" lg="12" xl="12" className="align-self-center pl-1 pr-1 pb-1">
```

Comment:
Hi @abrie, trying to do fine grained responsive layout with bootstrap is a non starter IMO. So I cannot really help you here. If you want to simply replace the `mitigationValue` `FastField` with a `NumericInput` I can give some input make the layout work.

Reply:
I've reverted the new layout to your version on master. I'm having a lot of trouble keeping track of how stuff works in there; some help integrating a NumericInput would be much appreciated.